internal/config/config.go

```diff
@@ -381,11 +376,33 @@ func fromFile(fileHandler file.Handler, name string) (*Config, error) {
 		if errors.Is(err, fs.ErrNotExist) {
 			return nil, fmt.Errorf("unable to find %s - use `constellation config generate` to generate it first", name)
 		}
+		if isAppClientIDError(err) {
+			return nil, UnsupportedAppRegistrationError{}
+		}
 		return nil, fmt.Errorf("could not load config from file %s: %w", name, err)
 	}
 	return &conf, nil
 }
 
+func isAppClientIDError(err error) bool {
```

Comment:
Do you think there could be a unittest that asserts there is only one such error? To prevent a situation in the future where this recognizes an unrelated key.